docs/tutorials/benchmarking/README.md

```diff
@@ -0,0 +1,17 @@
+# Benchmarking in Syne Tune
+
+
+This tutorial provides an overview of tooling which facilitates benchmarking of
+HPO algorithms in Syne Tune. The same tooling can also be used to rapidly
```

Comment:
remove "also" here : The same tooling can also be used ...

Reply:
OK